tests/unit/test__helpers.py

```diff
@@ -222,16 +217,19 @@ def raise_response():
         assert ret_val.status_code == status_codes[-1]
         assert status_codes[-1] not in common.RETRYABLE
 
-        assert func.call_count == 4
-        assert func.mock_calls == [mock.call()] * 4
+        assert func.call_count == 7
```

Comment:
Likewise, can we set the counts here (and below) using something like `len(status_codes)` (or adding / subtracting from it)?

Reply:
Good idea! Revised this test and other related tests in the file.